Makefile

```diff
@@ -167,22 +167,22 @@ $(OBJDIR)/%.o: %.rc $(HEADERS)
 
 install: $(TARGET)
 	rm -f $(OBJDIR)/git_version.o
-	mkdir -p $(DESTDIR)$(PREFIX)/bin 2>/dev/null || /bin/true
+	mkdir -p $(DESTDIR)$(BIN_DIR)/ 2>/dev/null || /bin/true
```

Comment:
Should `BIN_DIR` come with a default value? Initialized above, similar to `GLOBAL_CONFIG_DIR`